x-pack/plugins/apm/server/lib/event_metadata/get_event_metadata.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { QueryDslQueryContainer } from '@elastic/elasticsearch/api/types';
+import {
+  ERROR_ID,
+  SPAN_ID,
+  TRANSACTION_ID,
+} from '../../../common/elasticsearch_fieldnames';
+import { ProcessorEvent } from '../../../common/processor_event';
+import type { APMEventClient } from '../helpers/create_es_client/create_apm_event_client';
+
+export async function getEventMetadata({
```

Comment:
Would this be more accurate? 

```suggestion
export async function getDocumentFields({
```

Reply:
I think so, but the endpoint name is coupled to the UI component (intentionally, I think data type driven endpoints are not helpful, as they become too generic). So while it's more accurate, I think it might be better to consistently name this event metadata - or change everything including the endpoint name, which would not be my preference. WDYT?